app/controllers/chores_controller.rb

```diff
@@ -10,13 +12,25 @@ def show
   def new
   end
 
+  # def edit
+  #   @chore = Chore.find(params[:id])
+  # end
+
   def create
     @chore = Chore.new(chore_params)
 
     @chore.save
     redirect_to @chore
   end
 
+  def update
+    @chore = Chore.find(params[:id])
+    @chore.update!({:lastdone => Date.today,
```

Comment:
1. when referencing stackoverflow often you'll see this older syntax. You'll want to use `lastdone: Date.today` instead
2a. You should rename it so that it uses the underscore syntax. last_done instead of lastdone. 
2b. When doing this make sure you change the column name. Don't delete and add for this scenario